hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/NSSummaryEndpoint.java

```diff
@@ -62,6 +70,129 @@ public NSSummaryEndpoint(ReconNamespaceSummaryManager namespaceSummaryManager,
     this.reconSCM = reconSCM;
   }
 
+  @GET
+  @Path("/entitymetrics")
```

Comment:
uri name can be entityInfo or other better name, not a metrics as its not a metric data.

Reply:
@sumitagrawl - Thanks for reviewing. It has been fixed. Pls re-review.